std/selector/mux.go

```diff
@@ -14,7 +15,7 @@ import (
 // len(inputs) must be 2^len(selBits).
 func BinaryMux(api frontend.API, selBits, inputs []frontend.Variable) frontend.Variable {
 	if len(inputs) != 1<<len(selBits) {
-		panic("invalid input length for BinaryMux")
+		log.Panicf("invalid input length for BinaryMux (%d != 2^%d)", len(inputs), len(selBits))
```

Comment:
Same, use `panic`. If you want to format, then can use `panic(fmt.Sprintf(...))`